### DISCUSSION TITLE
Is there any option to give camleCase + Singular name when migrate from exiting data?

### DISCUSSION THREAD

**Discussion by minchjung** [prisma/prisma#14117] [Q&A] [answered]
Hello everyone 👋
I ’ve just picked up this wonderful library and wanted to migrate whole our backend service to the Prisma.
but I am a little struggling with mapping the name of model
after db-pull magically defines it’s name, field and everything
The thing is that our tables and fields on MySQL are currently following with the lower-case, snake-case and plural naming rule
What now I think its better to find another way rather than to write that code @@map on every single line (a lot to chang 😅  ) 
Plus
I’d like to use the class-generator and class-validator.
but that seemed to have the same problem ..
(generated class : Calmelcase, singluar
defined models: snake_case + lower_case, plurar name)
Is there any luck to save my life here please ? 🤣 

Following belows is what now we have
Tech: Express, Mysql, Sequealize,
Thank you for any help in advance 🙏 

class that prisma-class-generator created
<img width="814" alt="스크린샷 2022-07-03 오후 9 20 53" src="https://user-images.githubusercontent.com/78589751/177040980-437311e2-c27f-429c-8afa-be9904f2362c.png">

the model on prisma.schema after db pull (I'd like to map every name to use model as class)
<img width="1128" alt="스크린샷 2022-07-03 오후 9 21 13" src="https://user-images.githubusercontent.com/78589751/177041070-9c416e5d-c33b-4c45-b793-bab8604e4132.png">

**Comment by austincrim** [comment, marked as answer]
Hey @minchjung 👋,

`@@map` would not need to be done on every line, only on every `model`. Though renaming of fields doesn't **have** to happen at all, it is simply our recommendation.

**Reply by minchjung** [reply]
@austincrim  Thank you for your answer!! Yes. thats the recommendation but what I needed for that was to use class-generator and validator, but I found out that they're not necessary while they were in sequelize (with typescript). 
thanks anyway! 🙏

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
